chains/integration-tests/xcm/emulated/tests/assets/dancelight/src/tests/foreign_eth_token_to_container_chain_transfer.rs

```diff
@@ -0,0 +1,468 @@
+// Copyright (C) Moondance Labs Ltd.
+// This file is part of Tanssi.
+
+// Tanssi is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Tanssi is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Tanssi.  If not, see <http://www.gnu.org/licenses/>
+
+use {
+    container_chain_template_frontier_runtime::EthereumNetwork,
+    dancelight_emulated_chain::DancelightRelayPallet,
+    dancelight_runtime::SnowbridgeFeesAccount,
+    dancelight_system_emulated_network::{
+        DancelightRelay as Dancelight, DancelightSender, FrontierTemplatePara as FrontierTemplate,
+        FrontierTemplateSender, SimpleTemplatePara as SimpleTemplate, SimpleTemplateSender,
+    },
+    fp_account::AccountId20,
+    frame_support::traits::fungible::Mutate,
+    frame_support::{assert_ok, pallet_prelude::DispatchResult},
+    frontier_template_emulated_chain::FrontierTemplateParaPallet,
+    hex_literal::hex,
+    simple_template_emulated_chain::SimpleTemplateParaPallet,
+    snowbridge_beacon_primitives::{
+        types::deneb, AncestryProof, BeaconHeader, ExecutionProof, VersionedExecutionPayloadHeader,
+    },
+    snowbridge_core::ChannelId,
+    snowbridge_inbound_queue_primitives::{EventFixture, EventProof, Log, Proof},
+    sp_core::H160,
+    sp_core::U256,
+    sp_runtime::AccountId32,
+    sp_runtime::FixedU128,
+    xcm::latest::prelude::{Junctions::*, *},
+    xcm_emulator::{Chain, TestExt},
+};
+
+#[test]
+fn check_foreign_eth_token_to_frontier_container_chain_transfer_works() {
```

Comment:
Can we have also negative tests here or in the dancelight tests? something like:
- if fees account is 0, it does not work
- if token is not registered, it errors

I would also make sure not only the fees are deducted from fees account, but also that they end up in the sov account as well as the token